xla/service/gpu/matmul_utils_test.cc

```diff
@@ -225,6 +225,80 @@ TEST(GetMatrixLayoutTest, BatchInMostMinorPhysicalDimension) {
   EXPECT_FALSE(MatrixLayout::For(shape).ok());
 }
 
+using GetMatrixSizeRewriteThresholdTest = HloTestBase;
```

Comment:
Should this file now also have an additional test to make sure that the matrix multiplication is also rewritten if the data type is supported?

Reply:
I have added some tests, but I am not quite sure of their value.

Reply:
I think I would have checked that the matrix multiplication does not end up being codegen'd by old emitters if we expect it not to be covered correctly. I can make this change later to avoid the back and forth for you, since this seems good enough now :)